assembl/static2/js/app/components/debate/survey/post.jsx

```diff
@@ -110,63 +130,107 @@ class Post extends React.Component {
 
   render() {
     const { post } = this.props.data;
-    const { contentLocale, lang, screenWidth, originalLocale } = this.props;
+    if (!post.publicationState || post.publicationState.startsWith('DELETED')) {
+      return null;
```

Comment:
The ticket's requirements were to delete a post for phase1, but this component is used by all phases, is that not so?

Reply:
not this one (it's for survey only) but we already have post deletion for thread and multicolumn anyway